src/components/organisms/Room/Room.tsx

```diff
@@ -169,20 +171,20 @@ const Room: React.FC<RoomProps> = ({
     );
   });
 
-  const emptyComponents = [...Array(participants.length % 2)].map(
-    (e, index) => (
-      <div
-        key={`empty-participant-${index}`}
-        className={`participant-container-${capacity}`}
-      >
-        <img
-          className="empty-chair-image"
-          src="/empty-chair.png"
-          alt="empty chair"
-        />
-      </div>
-    )
-  );
+  const emptyComponents = !withoutChairs
```

Comment:
Not fan of the double negation. Can you call it `hasChair` of `showEmptySpaces` and default to true?